src/main/java/org/traccar/web/WebServer.java

```diff
@@ -172,6 +178,19 @@ private void initApi(Config config, ServletContextHandler servletHandler) {
     }
 
     private void initSessionConfig(Config config, ServletContextHandler servletHandler) {
+        boolean sessionPersisted = config.getBoolean(Keys.WEB_SESSION_PERSISTED);
+        if (sessionPersisted) {
+            DatabaseAdaptor databaseAdaptor = new DatabaseAdaptor();
+            databaseAdaptor.setDatasource(Context.getDataManager().getDataSource());
+            JDBCSessionDataStoreFactory jdbcSessionDataStoreFactory = new JDBCSessionDataStoreFactory();
+            jdbcSessionDataStoreFactory.setDatabaseAdaptor(databaseAdaptor);
+            SessionHandler sessionHandler = new SessionHandler();
+            SessionCache sessionCache = new DefaultSessionCache(sessionHandler);
+            sessionCache.setSessionDataStore(jdbcSessionDataStoreFactory.getSessionDataStore(sessionHandler));
+            sessionHandler.setSessionCache(sessionCache);
+            servletHandler.setSessionHandler(sessionHandler);
```

Comment:
What are the consequences of creating new `SessionHandler` and `SessionCache` instead of using default ones?

Reply:
the default SessionCache is null